modules/wpml/wpml-config.php

```diff
@@ -210,6 +214,124 @@ public function copy_term_metas( $metas, $sync ) {
 		return $metas;
 	}
 
+	/**
+	 * Adds post meta keys to export.
+	 *
+	 * @since 3.3
+	 * @see   PLL_Export_Metas
+	 *
+	 * @param  array $keys {
+	 *     A recursive array containing nested meta sub-keys to translate.
+	 *     Ex: array(
+	 *      'meta_to_translate_1' => 1,
+	 *      'meta_to_translate_2' => 1,
+	 *      'meta_to_translate_3' => array(
+	 *        'sub_key_to_translate_1' => 1,
+	 *        'sub_key_to_translate_2' => array(
+	 *             'sub_sub_key_to_translate_1' => 1,
+	 *         ),
+	 *      ),
+	 *    )
+	 * }
+	 * @return array
+	 *
+	 * @phpstan-param array<string, int|array> $keys
```

Comment:
```suggestion
	 * @phpstan-param array<string, int|string|array> $keys
```
I think @Chouby told me array value doesn't matter (except for sub key, which imply an array), and thus users can fill it with whatever the want.